### PR TITLE
fix bill_text template

### LINKED ISSUE #388: "transfer tax" search does not return this bill
[R2023-0004166](https://chicago.councilmatic.org/legislation/r2023-0004166/)

### PR DESCRIPTION
**Pull request by fgregg**
This had some out of date references which was causing the full text search to miss bills like in #388.

closes #388

I also added a management command to preview the content that will be search to the search enginge.

```
python manage.py preview_search_text R2023-0004166
```

Before the changes this is what would have been sent:

```python
{   'abstract': '',
    'actions': [   'R2023-0004166 action on 2023-09-14',
                   'R2023-0004166 action on 2023-09-14',
                   'R2023-0004166 action on 2023-11-01',
                   'R2023-0004166 action on 2023-11-01',
                   'R2023-0004166 action on 2023-11-07'],
    'bill_type': 'resolution',
    'bill_type_exact': None,
    'boost': 1.095890410958904,
    'controlling_body': ['Chicago City Council'],
    'controlling_body_exact': None,
    'description': 'Submission of public question by referendum to Chicago '
                   'voters for increase in Chicago real estate transfer tax '
                   'rates',
    'django_ct': 'chicago.chicagobill',
    'django_id': 'ocd-bill/722e9d40-9499-4a49-bb5d-8dd123866555',
    'friendly_name': 'Resolution R2023-0004166',
    'full_text': 'None',
    'id': 'ocd-bill/722e9d40-9499-4a49-bb5d-8dd123866555',
    'identifier': 'R2023-0004166',
    'inferred_status': 'Approved',
    'inferred_status_exact': None,
    'last_action_date': datetime.date(2023, 11, 7),
    'legislative_session': '2023',
    'legislative_session_exact': None,
    'ocr_full_text': 'None',
    'slug': 'r2023-0004166',
    'sort_name': 'ResolutionR2023-0004166',
    'sort_name_exact': None,
    'source_note': "['elms_api', 'web']",
    'source_url': "['https://api.chicityclerkelms.chicago.gov/matter/4C434D15-6452-EE11-BE6E-001DD8097692', "
                  "'https://chicityclerkelms.chicago.gov/Matter/?matterId=4C434D15-6452-EE11-BE6E-001DD8097692']",
    'sponsorships': ['Johnson, Brandon'],
    'sponsorships_exact': None,
    'text': '\n'
            'R2023-0004166\n'
            'Resolution R2023-0004166\n'
            '[&#x27;resolution&#x27;]\n'
            '\n'
            '\n'
            '\n'
            '  \n'
            '    Johnson, Brandon\n'
            '  \n'
            '\n'
            '\n'
            '    Chicago City Council\n'
            '    Introduced\n'
            '\n'
            '    Chicago City Council\n'
            '    Referred\n'
            '\n'
            '    Committee on Committees and Rules\n'
            '    Recommended to Pass\n'
            '\n'
            '    Chicago City Council\n'
            '    Deferred and Published\n'
            '\n'
            '    Chicago City Council\n'
            '    Passed\n'
            '\n'
            '\n'
            '\tNon-Routine\n'
            '\n'
            '\tCity Matters\n'
            '\n'
            '\tElections\n'
            '\n'
            'None\n',
    'topics': ['Non-Routine', 'City Matters', 'Elections'],
    'topics_exact': None}
```

and with the fixes, this what will be sent now:

```python
{   'actions': [   'R2023-0004166 action on 2023-09-14',
                   'R2023-0004166 action on 2023-09-14',
                   'R2023-0004166 action on 2023-11-01',
                   'R2023-0004166 action on 2023-11-01',
                   'R2023-0004166 action on 2023-11-07'],
    'bill_type': 'resolution',
    'bill_type_exact': None,
    'boost': 1.095890410958904,
    'controlling_body': ['Chicago City Council'],
    'controlling_body_exact': None,
    'django_ct': 'chicago.chicagobill',
    'django_id': 'ocd-bill/722e9d40-9499-4a49-bb5d-8dd123866555',
    'friendly_name': 'Resolution R2023-0004166',
    'full_text': 'None',
    'id': 'ocd-bill/722e9d40-9499-4a49-bb5d-8dd123866555',
    'identifier': 'R2023-0004166',
    'inferred_status': 'Approved',
    'inferred_status_exact': None,
    'last_action_date': datetime.date(2023, 11, 7),
    'legislative_session': '2023',
    'legislative_session_exact': None,
    'ocr_full_text': 'None',
    'slug': 'r2023-0004166',
    'sort_name': 'ResolutionR2023-0004166',
    'sort_name_exact': None,
    'source_note': "['elms_api', 'web']",
    'source_url': "['https://api.chicityclerkelms.chicago.gov/matter/4C434D15-6452-EE11-BE6E-001DD8097692', "
                  "'https://chicityclerkelms.chicago.gov/Matter/?matterId=4C434D15-6452-EE11-BE6E-001DD8097692']",
    'sponsorships': ['Johnson, Brandon'],
    'sponsorships_exact': None,
    'text': '\n'
            'Resolution R2023-0004166\n'
            '\n'
            '    R2023-0004166\n'
            '\n'
            '\n'
            'resolution\n'
            '\n'
            'Submission of public question by referendum to Chicago voters for '
            'increase in Chicago real estate transfer tax rates\n'
            '\n'
            '  \n'
            '    Johnson, Brandon\n'
            '  \n'
            '\n'
            '\n'
            '    Chicago City Council\n'
            '    Introduced\n'
            '\n'
            '    Chicago City Council\n'
            '    Referred\n'
            '\n'
            '    Committee on Committees and Rules\n'
            '    Recommended to Pass\n'
            '\n'
            '    Chicago City Council\n'
            '    Deferred and Published\n'
            '\n'
            '    Chicago City Council\n'
            '    Passed\n'
            '\n'
            '\n'
            '\tNon-Routine\n'
            '\n'
            '\tCity Matters\n'
            '\n'
            '\tElections\n'
            '\n'
            'None\n',
    'title': 'Submission of public question by referendum to Chicago voters '
             'for increase in Chicago real estate transfer tax rates',
    'topics': ['Non-Routine', 'City Matters', 'Elections'],
    'topics_exact': None}
```

### CODE REVIEW

File: chicago/templatetags/extras.py [hunk 1]
```diff
@@ -96,15 +96,6 @@ def clean_html(text):
     return re.sub(r"&(?:\w+|#\d+);", "", value)
 
 
-@register.filter
```
Comment (fgregg) [inline chicago/templatetags/extras.py:99]: this filter was only used in bill_text.txt and we aren't using it anymore.

File: chicago/search_indexes.py [hunk 2]
```diff
@@ -60,9 +57,6 @@ def prepare(self, obj):
 
         return data
 
-    def prepare_topics(self, obj):
```
Comment (fgregg) [inline chicago/search_indexes.py:63]: can get it more directly with model_attr

File: chicago/search_indexes.py [hunk 3]
```diff
@@ -15,18 +15,15 @@ class BillIndex(indexes.SearchIndex, indexes.Indexable):
...
     )
     slug = indexes.CharField(model_attr="slug", indexed=False)
     id = indexes.CharField(model_attr="id", indexed=False)
     bill_type = indexes.CharField(faceted=True)
     identifier = indexes.CharField(model_attr="identifier")
-    description = indexes.CharField(model_attr="title", boost=1.25)
+    title = indexes.CharField(model_attr="title", boost=1.25)
     source_url = indexes.CharField(model_attr="sources__url", indexed=False)
     source_note = indexes.CharField(model_attr="sources__note")
-    abstract = indexes.CharField(
```
Comment (fgregg) [inline chicago/search_indexes.py:27]: this is always empty for chicago.

**Review by derekeder — APPROVED** [review]
nice catch!

**Pull request merged**